ten_quick_tips_live_coding.md

```diff
@@ -53,46 +53,46 @@ university courses involving the teaching of programming or related techniques e
 
 There is a limited body of research on the effectiveness of Live coding in programming education.
 Most studies focus on non-participatory live coding,
-demonstrating the programming process and opposing this to the use of static code on slides
-(https://doi.org/10.1145/3279720.3279725 and references therein, as well as https://dl.acm.org/doi/10.1145/3373165.3373182).
-So far, the technique show that live-coding is as good as if not better than static code examples.
+demonstrating the programming process and contrasting this with the use of static code on slides
+(see https://doi.org/10.1145/3279720.3279725 and references therein, as well as https://dl.acm.org/doi/10.1145/3373165.3373182).
+So far, the technique show that live-coding is as good as if not better than using static code examples.
 
 The ten quick tips described below are aimed at those interested in applying the technique in to their own teaching.
-These complement the "Ten quick tips for delivering programming lessons"
+They are meant to complement the "Ten quick tips for delivering programming lessons"
 https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007433.
 
+
 ## Tip 1. Go slowly.
 
 Say out loud what you are doing while you do it
 for every command you type,
 every word of code you write,
-and every menu item or website button you click,
-and then point to the command and its output on the screen and optionally go through it a second time.
+and every menu item or website button you click.
+Then, point to the command and its output on the screen
+and optionally go through it a second time.
 This not only slows you down,
 it allows learners who are following along to copy what you do,
 or to catch up, even when they are looking at their screen while doing it.
 If the output of your command or code makes what you just typed disappear from view,
 scroll back up so learners can see it again.
 
-*Do not copy-paste code from your lesson script.*
-[FIXME: explain why not.]
+Do not copy-paste code from your lesson material and/or ask your learners to do the same. It will make you go fast with the risk of you not explaining the thought process behind the code.
```

Comment:
Replace second sentence with, "It's all too easy to go too fast without explaining the thought process behind the code. Typing everything in yourself reminds you just how much you are asking from the learners."

Reply:
+0.5